terraform/container_app_environment.tf

```
@@ -0,0 +1,73 @@
+resource "azurerm_container_app_environment" "backend" {
+  name                       = "cae-team4-backend-${var.environment}"
+  location                   = azurerm_resource_group.backend.location
+  resource_group_name        = "team4-backend"
+  log_analytics_workspace_id = azurerm_log_analytics_workspace.backend.id
+}
+
+resource "azurerm_log_analytics_workspace" "backend" {
+  name                = "law-team4-backend-${var.environment}"
+  location            = azurerm_resource_group.backend.location
+  resource_group_name = "team4-backend"
+  sku                 = "PerGB2018"
+  retention_in_days   = 30
+}
+
+resource "azurerm_container_app" "backend" {
+  name                         = "ca-team4-backend-${var.environment}"
+  container_app_environment_id = azurerm_container_app_environment.backend.id
+  resource_group_name          = "team4-backend"
+  revision_mode                = "Single"
+
+  identity {
+    type         = "UserAssigned"
+    identity_ids = [azurerm_user_assigned_identity.job_app_backend.id]
+  }
+
+  registry {
+    server   = data.azurerm_container_registry.acr.login_server
+    identity = azurerm_user_assigned_identity.job_app_backend.id
+  }
+
+  template {
+    container {
+      name   = "backend"
+      image  = "${data.azurerm_container_registry.acr.login_server}/job-app-backend:latest"
```

```suggestion
      image  = "${data.azurerm_container_registry.acr.login_server}/job-app-backend:latest"
```

job-app-backend doesn't exist in [ACR](https://portal.azure.com/?l=en.en-gb#@kainosai.onmicrosoft.com/resource/subscriptions/ef1b41f6-e4b2-41d0-a52d-d279af4a77ab/resourceGroups/container-registry/providers/Microsoft.ContainerRegistry/registries/aiacademy25/repository)